server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -2213,13 +2214,75 @@ public NetworkResponse createNetworkResponse(ResponseView view, Network network)
                         CapabilityResponse capabilityResponse = new CapabilityResponse();
                         String capabilityValue = ser_cap_entries.getValue();
                         capabilityResponse.setName(capability.getName());
-                        capabilityResponse.setValue(capabilityValue);
+                        if (Service.Lb == service && capability.getName().equals(Capability.SupportedLBIsolation.getName())) {
+                             capabilityResponse.setValue(networkOffering.isDedicatedLB() ? "dedicated" : "shared");
+                        } else {
+                            capabilityResponse.setValue(capabilityValue);
+                        }
+                        if (capability.getName().equals(Capability.SupportedLBIsolation.getName()) || capability.getName().equals(Capability.SupportedSourceNatTypes.getName())
```

Comment:
Maybe an alternative would be creating a variable in this class defining a Set of Strings that fit in this conditional.
This would look something similar to:
```
private final Set<String> <insertNameHere> = new HashSet<>(Arrays.asList(Capability.SupportedLBIsolation.getName(),
                                                                          Capability.SupportedSourceNatTypes.getName(),
                                                                          Capability.RedundantRouter.getName()));
...
...
...
     if (<insertNameHere>.contains(capability.getName())) {
          capabilityResponse.setCanChoose(true);
     }
...
...
```